ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -1010,5 +998,43 @@ public void saveAfterTime() {
     public void cleanDownloadFolder() throws IOException {
         CustomWebDriverProvider.cleanDownloadFolder();
     }
+
+    private enum Credentials {
+        DROPBOX,
+        SALESFORCE,
+        TWITTER,
+        OPENSHIFT,
+        SYNDESIS,
+        SLACK,
+        S3,
+        FTP,
+        GOOGLE_MAIL,
+        GOOGLE_SHEETS,
+        GOOGLE_CALENDAR,
+        SAP_CONCUR,
+        AWS,
+        TELEGRAM,
+        SERVICENOW,
+        BOX,
+        SEND_EMAIL_SMTP,
+        RECEIVE_EMAIL_IMAP_OR_POP3,
+        GITHUB,
+        ZENHUB,
+        JIRA_HOOK,
+        AMQ,
+        AMQP,
+        MQTT,
+        AWS_DDB
+    }
+
+    private static class Element {
+        public static final By LOGIN_BUTTON = By.className("btn");
+        public static final By NAVIGATION_PANEL = By.className("pf-c-nav");
+        public static final By NAVIGATION_USER_DROPDOWN = By.cssSelector("[data-testid=app-top-menu-user-dropdown]");
+
+        public static final By navigationLink(String title) {
+            return By.cssSelector(String.format("a[data-testid=\"ui-%s\"]", title.toLowerCase().replaceAll("[\\s_]", "-")));
+        }
+    }
```

Comment:
This might be a misconfiguration in your IDE, should not happen

Reply:
Don't think I've enabled or recofigured something like that in my IDE, but it seemed strange to me as well. Have to go thru settings and see if I can switch this behavior off.